integration-tests/tests/cli/schema.spec.ts

```
@@ -453,3 +453,47 @@ test.concurrent(
     `);
   },
 );
+
+test.only('schema:check gives correct error message for missing `--service` name flag in federation project', async ({
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)